src/components/Card.jsx

```diff
@@ -1,64 +1,69 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 import { connect } from 'react-redux'
 import { likeOrDislike, toggleSignInModal } from '../actions/actionCreators'
 
 const whiteBackground = {
   backgroundColor: '#FFF'
-}
+};
 
-class Card extends React.Component {
-  state = {
-    liked: false
-  }
+const Card = ({
+  props: {
+    dispatch,
+    home: {
+      loggedIn,
+    },
+    index,
+    project,
+  },
+}) => {
+  const [liked, setLiked] = useState(false);
 
-  handleCick = (index) => {
-    const { loggedIn } = this.props.home
+  useEffect(() => {
```

Comment:
I am just starting to learn useEffect, but from the docs it looks like it runs after every render and update. We wouldnt want to dispatch an action after every render but only when the user clicks the the button.